weaviate/gql/filter.py

```diff
@@ -25,6 +25,23 @@
     "valueGeoRange",
 }
 
+WHERE_OPERATORS = [
+    "And",
+    "ContainsAll",
+    "ContainsAny",
+    "Equal",
+    "GreaterThan",
+    "GreaterThanEqual",
+    "IsNull",
+    "LessThan",
+    "LessThanEqual",
+    "Like",
+    "Not",
```

Comment:
this one is gone

Reply:
What do you mean exactly?

Reply:
"Not" does not exist anymore. It never worked in the first place, so we removed it from weaviate

Reply:
Ahh, okay! Will remove